scripts/setup_gradient_workflow.py

```
@@ -0,0 +1,234 @@
+    """Manages Paperspace Gradient workflow setup and execution."""
+
+    def __init__(self, workflow_file: str = ".gradient/workflows/training-pipeline.yaml"):
+        self.workflow_file = Path(workflow_file)
+        self.api_key = os.getenv("PAPERSPACE_API_KEY")
+        self.project_id = os.getenv("PAPERSPACE_PROJECT_ID")
+
+    @staticmethod
+    def check_gradient_cli() -> bool:
+        """Check if gradient CLI is available."""
+        try:
+            result = subprocess.run(["gradient", "--version"], 
```

Ruff (0.12.2)

Trailing whitespace

Remove trailing whitespace

(W291)